clients/benchmarks/CMakeLists.txt

```diff
@@ -232,8 +232,50 @@ endif()
 
 if (NOT WIN32)
   if (BUILD_WITH_ROCTX)
-    target_link_libraries(rocsparse-bench PRIVATE -lroctx64)
-    target_compile_definitions( rocsparse-bench PRIVATE ROCSPARSE_CLIENTS_WITH_ROCTX )
+
+    set(ROCTX_PATH "" CACHE STRING "Path to the roctx library (directory containing libroctx64.so)")
+
+    find_path(ROCTRACER_INCLUDE_DIR
+      NAMES roctracer/roctx.h
+      HINTS
+      ${ROCTX_PATH}
+      ${ROCTX_PATH}/include
+      /opt/rocm/include
+      ${ROCTX_PATH}/../include
+      DOC "Path to the roctracer include directory containing roctx.h")
+
+    find_library(ROCTX_LIBRARY
+      NAMES roctx64
+      HINTS
+      ${ROCTX_PATH} # User-provided path
+      ${ROCTX_PATH}/lib
+      PATHS
+      /opt/rocm/lib # Default ROCm path
+      NO_DEFAULT_PATH
+    )
+
+    if(ROCTRACER_INCLUDE_DIR AND ROCTX_LIBRARY)
+      message(STATUS "Found roctracer include directory: ${ROCTRACER_INCLUDE_DIR}")
+      message(STATUS "Found roctx library: ${ROCTX_LIBRARY}")
+
+      # Add the include directory
+      target_include_directories(rocsparse-bench PRIVATE ${ROCTRACER_INCLUDE_DIR})
+
+      # Link the rocTx lib
+      target_link_libraries(rocsparse-bench PRIVATE ${ROCTX_LIBRARY})
+      target_compile_definitions( rocsparse-bench PRIVATE ROCSPARSE_CLIENTS_WITH_ROCTX )
+      message(STATUS "ROCTX tracing support enabled for target rocsparse-bench.")
+    else()
+      message(WARNING "ROCTX tracing will be disabled for target rocsparse-bench.")
+
```

Comment:
Maybe put this message after the two warnings below, as a consequence of not finding the header and/or the library. 
I think ROCTX shouldn't be disabled because we don't find what's needed. If ROCTX is requested, then we should return an error in the situation where the header or the library is not found. 

This code is in the section "if (BUILD_WITH_ROCTX)", thus we should add the token 'REQUIRED' to find_path and find_library, and skip the condition "if(ROCTRACER_INCLUDE_DIR AND ROCTX_LIBRARY)".